mgmt_cli_test.py

```diff
@@ -174,7 +178,8 @@ def get_table_id(self, node, table_name, keyspace_name=None, remove_hyphen=True)
             return base_id.replace('-', '')
         return base_id
 
-    def restore_backup(self, mgr_cluster, snapshot_tag, keyspace_and_table_list):  # pylint: disable=too-many-locals
+    def restore_backup_outside_of_manager(self, mgr_cluster, snapshot_tag, ks_tables_list, location=None,
```

Comment:
I think this method requires docstring - telling what it does, limitations etc. 
It's not obvious for 'outside' users to understand what 'outside_of_manager' means (for me it would be better `restore_backup_without_manager`)
Please also add link to official scylla docs on this procedure.

Reply:
Agree, will fix it